src/Tuura/Concept/STG/Translation.hs

```diff
@@ -69,7 +69,7 @@ transition (f, t)
         | otherwise  = readArc (init (show f) ++ "0") t
 
 tmpl :: String
-tmpl = unlines [".model out", ".inputs %s", ".outputs %s", ".internal %s", ".graph", "%s.marking {%s}", ".end", "%s"]
+tmpl = unlines [".model out", ".inputs %s", ".outputs %s", ".internal %s", ".graph", "%s.marking {%s}", "%s.end"]
```

Comment:
Having two elements add's an unnecessary new line. 

I can remove this by adding a check somewhere for the last element in a list, but that seems like a lot more code.